tests/unit/test-firefox/test.remote.js

```diff
@@ -375,4 +376,20 @@ describe('firefox.remote', () => {
 
   });
 
+  describe('defaultRemotePortFinder', () => {
+
+    function findRemotePort() {
```

Comment:
`findRemotePort` doesn't seem to be needed, we should just call `findFreeTcpPort` directly in the test cases.